content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -87,6 +87,85 @@ Instead of immediately awaiting the result of each operation when it's performed
 >
 > Refer to the 📖 [transactions guide](../../../guides/performance-and-optimization/prisma-client-transactions-guide#transaction-api) for more examples.
 
+### Interactive Transactions
+
+Sometimes you need more control over what queries execute within a transaction.
+
+Interactive transactions are commonly used when the result of one query depends on another query and you want to run both queries within a single transaction.
+
+To support this use case and more, you can pass an async function into `$transaction`.
+
+The following is an example of purchasing a movie ticket online. This is a good use case for an interactive transactions because you don't want to sell the same ticket twice.
```

Comment:
```suggestion
The following is an example of purchasing a movie ticket online. This is a good use case for interactive transactions because you don't want to sell the same ticket twice.
```